js/index.js

```diff
@@ -40,3 +40,61 @@ const siteContent = {
 // Example: Update the img src for the logo
 let logo = document.getElementById("logo-img");
 logo.setAttribute('src', siteContent["nav"]["img-src"])
+
+const navLinks = document.querySelectorAll('header nav a');
+navLinks[0].innerHTML = siteContent.nav["nav-item-1"]
+navLinks[1].innerHTML = siteContent.nav["nav-item-2"]
+navLinks[2].innerHTML = siteContent.nav["nav-item-3"]
+navLinks[3].innerHTML = siteContent.nav["nav-item-4"]
+navLinks[4].innerHTML = siteContent.nav["nav-item-5"]
+navLinks[5].innerHTML = siteContent.nav["nav-item-6"]
+
```

Comment:
You can use a loop for this, example: 
`for (let i = 0; i < navItems.length; i++) { navItems[i].textContent = siteContent.nav[nav-item-${i + 1}];`